Sources/CloudyKit/URLSession+Helpers.swift

```diff
@@ -143,7 +147,10 @@ extension NetworkSession {
                 let dictionaries = value.map { CKWSReferenceDictionary(recordName: $0.recordID.recordName, action: $0.action.stringValue) }
                 fields[fieldName] = CKWSRecordFieldValue(value: .referenceList(dictionaries), type: nil)
             default:
-                fatalError("unable to handle \(value) of type \(type(of: value))")
```

Comment:
Isn't it better for the user of the API to get this as a `fatalError`. The description could probably be improved just to mention that it's not supported.

Reply:
Hey @camdenfullmer thanks for checking, I'm also a bit out of time now but will take a look as soon as I can ;)